src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -203,6 +222,18 @@ class Container extends React.Component<Props, ComponentState> {
         }
     }
 
+    areEnumsIdentical(newEnums: CLM.EnumValue[], oldEnums: CLM.EnumValue[]): boolean {
+        // If any new enums, or old ones changed or deleted
+        if (newEnums.every(ev => ev.enumValueId !== undefined) &&
+            oldEnums.every(oldEnum => {
+            let newEnum = newEnums.find(ne => ne.enumValueId === oldEnum.enumValueId)
+            return (newEnum !== undefined && newEnum.enumValue === oldEnum.enumValue)
+        })) {
+            return true
+        }
+        return false
```

Comment:
Do we need the outer `if`?

`every` already returns `true` or `false` and the `&&` ensures both so the outer `if` seems unnecessary.

Reply:
good catch